src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -175,12 +176,13 @@ export default {
 			}
 		},
 
-		isReadOnly() {
+		disabled() {
 			return this.conversation.readOnly === CONVERSATION.STATE.READ_ONLY
+			|| !this.currentConversationIsJoined
```

Comment:
```suggestion
		disabledActions() {
			return this.conversation.readOnly === CONVERSATION.STATE.READ_ONLY
				|| !this.currentConversationIsJoined
```

Would rename the computed to reflect that only parts are disabled and not the complete component

Reply:
Aren't all the parts of the component (Actions + iIput field) disabled?

Reply:
Did you want to allow writing messages already and just not sending them?

Reply:
Not at this stage we said